### DISCUSSION TITLE
Creating a bundle: examples / docs?

### DISCUSSION THREAD

**Discussion by automatix** [akeneo/pim-community-dev#20474] [General]
Hello guys!

I'm developing a bundle for Akeneo PIM Community Edition v5. Technically a Symfony Bundle, that is stored externally in a separate repo and is added to the project via Composer. Now I'm having some troubles to integrate it into the Akeneo application. Especially extending of views (in my concrete case the product list and the product details view is problematic.)

The Akeneo documentation is very, vary... slim. :) There is the article "[Create a reusable bundle](https://docs.akeneo.com/5.0/technical_architecture/best_practices/reusable_bundle.html)", that provides some advices and warnings. Which is useful. But it doesn' cover the topic. And that is all I could find...

Are there any how-tos? Are there examples (especially for the v5)?

Thx

**Comment by devwitty409** [comment]
Hi dear,
Nice to meet you.
I came across your project and wanted to share my experience—I’ve successfully built a nearly identical solution for Akeneo PIM.

In a recent project, I developed a custom Symfony bundle to automate cloud storage integration (AWS S3) with Akeneo, handling:
Event-based uploads during product saves/imports
Bulk file processing for API imports
Dynamic path overrides to serve images directly from cloud storage

The solution reduced the client’s local storage costs by 60% while maintaining seamless UI/API workflows.
For your Google Cloud Storage needs, I can deliver:

Let me know if you’d like to discuss specifics—I’d be happy to provide code snippets or a demo.

Best regards.
